.jenkins/Jenkinsfile-s390x

```diff
@@ -0,0 +1,255 @@
+                }
+            }
+        }
+        stage('Execute system tests') {
+            steps {
+                script {
+                    catchError {
+                        // For upgrade tests running with locally built images we need to change tag in StrimziUpgradeST.json
+                        if ("${env.TEST_PROFILE}".contains("upgrade")) {
+                            println("[INFO] Update files for upgrade procedure")
+                            sh(script: """
+                                sed -i 's#:latest#:${env.DOCKER_TAG}#g' ${env.WORKSPACE}/systemtest/src/test/resources/upgrade/StrimziUpgradeST.json ${env.WORKSPACE}/install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml
+                                sed -i 's#quay.io/strimzi/test-client:${env.DOCKER_TAG}#${env.DOCKER_REGISTRY}/strimzi/test-client:${env.DOCKER_TAG}#g' ${env.WORKSPACE}/systemtest/src/test/resources/upgrade/StrimziUpgradeST.json
+                                sed -i 's#quay.io/strimzi/#${env.DOCKER_REGISTRY}/strimzi/#g' ${env.WORKSPACE}/install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml
+                                sed -i 's#/opt/${env.DOCKER_REGISTRY}#/opt#g' ${env.WORKSPACE}/install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml
+                            """)
+                            sh(script: "cat ${env.WORKSPACE}/systemtest/src/test/resources/upgrade/StrimziUpgradeST.json")
+                            sh(script: "cat ${env.WORKSPACE}/install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml")
+                        }
+                        lib.runSystemTests(env.WORKSPACE, env.TEST_CASE, env.TEST_PROFILE, env.TEST_GROUPS, env.EXCLUDE_GROUPS, env.TEST_COUNT_RUNNING_IN_PARALLEL)
+                    }
+                }
+            }
+        }
+    }
+    post {
+        always {
+            script {
+                sh(script: "minikube stop || true")
+                sh(script: "minikube delete || true")
+                sh(script: "docker stop s390x-registry || true")
+                sh(script: "docker rm s390x-registry || true")
+                sh(script: "sudo ./systemtest/scripts/results_info.sh ./systemtest/target/failsafe-reports/ ${env.TEST_CASE} ${env.TEST_PROFILE} ${env.BUILD_NUMBER} ${env.KUBE_VERSION} ${env.TEST_ONLY} ${env.TEST_COUNT_RUNNING_IN_PARALLEL} ${env.EXCLUDE_GROUPS} ${env.ENV_VARS}")
+                //lib.postGithubPrComment("results.json")
+                lib.postAction(env.ARTIFACTS_DIR, env.ghprbPullId, env.ghprbActualCommitAuthor, env.ghprbPullTitle, env.ghprbPullLink, env.BUILD_URL, env.WORKSPACE, "mail not enabled")
+            }
+        }
+        failure {
+            println("Build failed")
+            script {
+                lib.sendMail(env.STRIMZI_MAILING_LIST, "failed", env.ghprbPullId, env.ghprbActualCommitAuthor, env.ghprbPullTitle, env.ghprbPullLink, env.BUILD_URL)
```

Comment:
Don't forget to set `STRIMZI_MAILING_LIST` env in your Jenkins, otherwise it will fail on it I think